tools/sdk/libc/xtensa-lx106-elf/include/locale.h

```diff
@@ -1,7 +1,7 @@
 /*
 	locale.h
 	Values appropriate for the formatting of monetary and other
-	numeric quantities.
+	numberic quantities.
```

Comment:
```suggestion
	numeric quantities.
```

Reply:
I can do this, but the problem stems from the newlib itself, from where this file is copied.  Doing the change in the newlib repo is easy, but it's unfortunately very hard to get igrr's time to look at PRs and merge them there.

For something like this, I'd rather not bother or bite the bullet and say we're using newlib from another repo (I can share commit permissions on mine, make a new one somewhere, etc.) and include it as an SDK directory module (like lwip and bssl).

Reply:
Personally I like include as module - the less "binary" includes the better.

Reply:
It still wouldn't compile from the Arduino environment.  You **really** wouldn't want to build libc single-threaded every application, believe me!  But it would be kind of like BearSSL where there'd be a `make install` target that would automate updates to the core .a dirs.

Reply:
About the initial trivial fix, I just experimented the inline PR beta github feature. Sorry for the noise.